README.md

```diff
@@ -169,6 +169,8 @@ git clone https://github.com/chipsalliance/chisel3.git
 cd chisel3
 sbt compile
 ```
+In order to run the following unit tests, make sure you have Verilator installed and loaded
+(Check this by running 'which verilator')
```

Comment:
```suggestion

In order to run the following unit tests, make sure you have Verilator installed and loaded
(you can check this by running 'which verilator').
```

Reply:
```suggestion
In order to run the following unit tests, make sure you have Verilator installed and on your `PATH`
(Check this by running `which verilator`).
```

Reply:
I like Jack's suggestion better

Reply:
i think they can both be used together; i'm just suggesting that the capitalization be fixed in the parentheses.

Reply:
I've incorporated your change into mine.